tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -449,6 +449,14 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string RemoveClientServiceNamesEnabled = "DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED";
 
+        /// <summary>
+        /// Configuration key for the comma-separated list of user disabled
+        /// ADO.NET CommandType names that should not have Span created for them.
+        /// <para>"InterceptableDbCommand" and "ProfiledDbCommand" are always disabled by default.</para>
+        /// </summary>
+        /// <seealso cref="TracerSettings.DisabledAdoNetCommandTypes"/>
+        public const string DisabledAdoNetCommandTypes = "DD_TRACE_DISABLED_ADONET_COMMAND_TYPES";
```

Comment:
Must add this to the config remapper

Reply:
Need to add it manually here in the repo and make a PR

Reply:
https://github.com/DataDog/dd-go/pull/150493

Reply:
updated the file manually with: `"DD_TRACE_DISABLED_ADONET_COMMAND_TYPES": "trace_disabled_adonet_command_types",`